http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/QueueFactory.scala

```diff
@@ -17,4 +17,6 @@ object QueueFactory {
     Queue.bounded[IO, Event[LeoPubsubMessage]](queueSize).unsafeRunSync()
 
   def asyncTaskQueue() = Queue.bounded[IO, Task[IO]](queueSize).unsafeRunSync()
+
+  def makeDateAccessedQueue() = Queue.bounded[IO, UpdateDateAccessMessage](10).unsafeRunSync()
```

Comment:
What does the `10` stand for here, no need to change anything, just curious :)

Reply:
The size allocated for the queue in memory